.github/workflows/release.yaml

```
@@ -19,9 +13,26 @@ jobs:
 
   release:
     needs: check
-    uses: canonical/bootstack-actions/.github/workflows/charm-release.yaml@v3
-    secrets: inherit
-    with:
-      channel: "latest/edge"
-      upload-image: false
-      runs-on: "['ubuntu-20.04']"
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+      - name: Initialize lxd  # This should dropped once it's implemented on charming-actions itself. https://github.com/canonical/charming-actions/issues/140
+        if: ${{ inputs.destructive-mode == false }}
+        run: |
+          sudo lxd init --auto
+          # The following is needed to fix network error when creating new base instance on lxd
+          sudo chmod a+wr /var/snap/lxd/common/lxd/unix.socket
+          lxc network set lxdbr0 ipv6.address none
+          sudo usermod -a -G lxd $USER
+          if [[ $(cat /etc/os-release | grep VERSION_CODENAME) == *"jammy"* ]]; then
```

nit: useless use of cat
```suggestion
          if grep -q CODENAME=jammy /etc/os-release; then
```